manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -4,7 +4,7 @@ images:
   argoworkflowcontroller: gcr.io/ml-pipeline/workflow-controller:v2.3.0
   cloudsqlproxy: gcr.io/cloudsql-docker/gce-proxy:1.14
   frontend: gcr.io/ml-pipeline/frontend:0.1.27
-  metadataserver: gcr.io/kubeflow-images-public/metadata:v0.1.8
+  metadataserver: gcr.io/tfx-oss-public/ml_metadata_store_server:0.14.0
```

Comment:
Could you update here too https://github.com/kubeflow/pipelines/blob/master/.release.cloudbuild.yaml#L130

Reply:
Ack.

Reply:
Done